test/include/test_runner/test_group.h

```diff
@@ -18,6 +20,9 @@ struct TestStatement {
     bool enumerate = false;
     bool checkOutputOrder = false;
     std::string expectedTuplesCSVFile;
+    // for multiple conns
+    std::optional<std::string> conn_name; // default conntion name conn0
```

Comment:
ditto. camel case. And I believe default name is "default"? Replace default with a constant and apply across this PR

Reply:
@hououou did you address this comment?